src/Components/Nav/Nav_Components/NavDropDown.js

```diff
@@ -35,62 +34,75 @@ const category = {
   ],
 };
 
-function NavDropDown() {
+function NavDropDown({ hide, hideToggle, visible }) {
   return (
-    <DropDown>
+    <DropDown hide={hide} visible={visible}>
       {Object.keys(category).map((title) => {
         return <Category title={title} list={category[title]} />;
       })}
+      <CloseDropBtn
+        onClick={hideToggle}
+        alt="close button"
+        src="/Images/main_images/close.webp"
+      />
     </DropDown>
   );
 }
 
-const moveDown = keyframes`
-  0% {
-    top: -400px;
-  100% {
-    top: -75px;
-  }
-`;
-
-const moveUp = keyframes`
-  0% {
-    top: -75px;
-  100% {
-    top: -400px;
-  }
-`;
-
 const DropDown = styled.div`
-  position: relative;
+  position: fixed;
   display: flex;
   justify-content: center;
-  animation: ${moveDown} 1s ease-out;
-  animation-direction: alternate;
-  animation-fill-mode: forwards;
-  top: -75px;
+  transition-duration: 1s;
   padding-top: 80px;
   width: 100vw;
-  height: 250px;
+  height: ${({ visible }) => (visible ? "360px" : "330px")};
   background-color: ${theme.darkGrey};
-  z-index: 2;
+  z-index: 800;
+  ${({ hide, visible }) => {
+    if (visible) {
+      if (hide) {
+        return css`
+          top: -400px;
+        `;
+      } else {
+        return css`
+          top: 0px;
+        `;
+      }
+    } else {
+      if (hide) {
+        return css`
+          top: -450px;
+        `;
+      } else {
+        return css`
+          top: 0px;
+        `;
+      }
+    }
+  }};
```

Comment:
```suggestion
  z-index: 800;
  top: ${({ visible, hide }) => !hide ? 0 : visible ? -400 : -450}px;
```
이렇게 줄여주세요. 결국 0이거나 -400, -450이 리턴될 것이라는 사실에 착안하면 더 잘 보입니다

Reply:
우와 대박 감사합니다! 넘 좋은 코드예요